docs/agent-oriented-approach.md

```diff
@@ -0,0 +1,49 @@
+
+It is important to emphasise that by decentralisation we do not mean distribution; although multi-agent systems typically do tend to be distributed. A distributed system is one whose components are physically located in different places and connected over a network. A fully centralised system, owned and operated by a single entity, may in fact be highly distributed. Google's infrastructure is an example of this, where all of the components are distributed across the globe, yet designed to work together highly efficiently, and function in unison. Decentralisation on the other hand refers to a system whose components may be owned, operated, and managed by different stakeholders, each with their own personal objectives, interests, and preferences, which may not necessarily be aligned with one another or that of the system itself. Therefore, distribution refers to the physical placement of a system's components, whereas decentralisation refers to a) the diversity of ownership and control over a system's constituents, and b) the absence of a central point of authority between them.    
+
+To better illustrate the distinction between centralised and decentralised systems, consider search and discoverability in a commerce environment. In a central system (say amazon.com), there is a single search service provider -- owned and run by the commerce company itself -- which takes care of all search related functionality for every product within their domain. So to be discoverable in this system, all sellers must register their products with this particular search service provider. However in a decentralised system, there may not necessarily be a single search service provider. There may be multiple such providers, run by different, perhaps competing entities. Each seller has the freedom to register with (i.e. make themselves known to) one or a handful of providers. On the buyers side, the more providers they contact and query, the higher their chances of finding the product they are looking for. 
```

Comment:
I think the case is already quite clear and adding another example makes the text very long.
```suggestion
```